deploy/kubernetes/elastic-agent-kustomize/default/elastic-agent-standalone/base/elastic-agent-standalone-daemonset.yaml

```diff
@@ -40,7 +40,7 @@ spec:
       #        mountPath: /etc/elastic-agent/inputs.d
       containers:
         - name: elastic-agent-standalone
-          image: docker.elastic.co/elastic-agent/elastic-agent:9.1.0
+          image: docker.elastic.co/elastic-agent/elastic-otel-collector:9.1.0
```

Comment:
should we change this one to the otel-collector? 🤔 this is a k8s manifest of the "traditional" elastic-agent